client/app/dbaas/logs/detail/inputs/add/configure/logs-inputs-add-configure.controller.js

```diff
@@ -45,8 +45,18 @@ class LogsInputsAddConfigureCtrl {
     });
     this.previousTest = this.test;
 
+    this.engine = this.CucControllerHelper.request.getHashLoader({
```

Comment:
We should try to avoid using `CucControllerHelper` I think and simply use promises

Reply:
This is done. Thanks